scripts/bin/lint.ts

```diff
@@ -9,7 +9,7 @@ async function main() {
   const PKG = await promptForPackage();
 
   printSeparator('Linting TypeScripts');
-  await execa('yarn', ['wsrun', '--serial', 'eslint', '--fix', '.'], {
+  await execa('yarn', ['wsrun', '--serial', 'eslint', '--fix'], {
```

Comment:
Does this cancel the linter stage?

Reply:
I changed this to fix below typescript-eslint issue (explained below)